src/Splat.Microsoft.Extensions.DependencyInjection/MicrosoftDependencyResolver.cs

```diff
@@ -196,7 +191,7 @@ public virtual void UnregisterCurrent(Type? serviceType, string? contract = null
     /// ignoring the <paramref name="serviceType"/> argument.
     /// </summary>
     /// <param name="serviceType">The service type to unregister.</param>
-    /// <param name="contract">This parameter is ignored. Service will be removed from all contracts.</param>
+    /// <param name="contract">A optional value which will remove only an object registered with the same contract.</param>
```

Comment:
@glennawatson can you just sanity check\ ok this. not sure if previous documentation is hang up of an older implementation, or it's the actual intended behaviour?

Reply:
I would agree that when the contract is provided we only remove that instance, if no contract is provided, remove all.
Glenn's view on this would also be good.